packages/playground/src/components/node_selector/TfAutoNodeSelector.vue

```diff
@@ -185,6 +186,8 @@ export default {
       required: true,
     },
     nodesLock: Object as PropType<AwaitLock>,
+    loadFarm: { type: Function as PropType<(farmId: number) => Promise<Farm | undefined>>, required: true },
+    getFarm: { type: Function as PropType<(farmId: number) => Farm | undefined>, required: true },
```

Comment:
What is the difference between `load farm` and `get farm`?

Reply:
shown nodes is resulted as reactive value from computed so I avoid any async calls while validating as regular vue computed prop not allowing async functions

loadFarm => making sure farm is loaded in farms map (async)
getFarm => get farm from farms map (sync) if found